frontend/src/Editor/Editor.jsx

```diff
@@ -387,19 +400,13 @@ class EditorComponent extends React.Component {
       if (version?.id === this.state.app?.current_version_id) {
         (this.canUndo = false), (this.canRedo = false);
       }
-      useAppVersionStore.getState().actions.updateEditingVersion(version);
```

Comment:
@johnsoncherian we"ll need useAppVersionStore.getState().actions.updateEditingVersion(version) instead of useAppDataStore.getState().actions.updateEditingVersion(version);